cmd/project.go

```diff
@@ -107,7 +108,21 @@ func ProjectCommand() cli.Command {
 func projectLs(ctx *cli.Context) error {
 	c, err := GetClient(ctx)
 	if err != nil {
-		return err
+		path := ctx.GlobalString("cf")
```

Comment:
The fix should be moved into GetClient as this is going to affect anything that calls it. The fix should not rewrite the config file as we don't know what is causing the 404. The error coming back should be checked and if it is a 404 from the project, extend the error and provide info on how to select another project - either context switch if possible or login